tools/syz-declextract/README.md

```diff
@@ -35,3 +35,13 @@ make -j`nproc` syz-declextract
 ```
 ./bin/syz-declextract $KERNEL/fs/read_write.c | less # or any other .c file
 ```
+## Running the tool
+Add the binary to your $PATH
+```
+export PATH=$PATH:$SYZ/bin
```

Comment:
I think it's better to pass the binary as a flag.
Check out the standard `flag` package. We will probably need more flags in future anyway.

Reply:
Done. Also passed the compilation database as a flag.